tests/gradients/core/test_pulse_gradient.py

```diff
@@ -1306,7 +1306,7 @@ def test_shots_attribute(self, dev_name, shots):  # pylint:disable=unused-argume
 @pytest.mark.jax
 @pytest.mark.parametrize("dev_name", ["default.qubit", "default.qubit.jax"])
 class TestStochPulseGradQNode:
-    """Test that pulse_generator integrates correctly with QNodes."""
+    """Test that pulse_odegen integrates correctly with QNodes."""
```

Comment:
```suggestion
    """Test that stoch_pulse_grad integrates correctly with QNodes."""
```
I _think_, no?

Reply:
yup there were some places where stoch_pulse_grad stuff has pulse_generator in the doc strings, probably from copy-pasting? Nice catch! :)